README.md

```diff
@@ -104,7 +104,7 @@ See `DocGeneratorRunner` for examples of how to configure builders for each of t
 
  * All implementations support the following methods `createTable`, `describeTable`, `deleteTable`, `getItem`, `batchGetItem`, `putItem`, `scan`, and `query`.
  * The following methods are NOT supported: `updateTable`, `batchWriteItem`, `createBackup`, `deleteBackup`, `listBackups`, `restoreTableFromBackup`, `createGlobalTable`, `updateGlobalTable`, `describeGlobalTable`, `listGlobalTables`, `describeContinuousBackups`, `describeLimits`, `describeTimeToLive`, `updateTimeToLive`, `listTagsOfResource`, `tagResource`, `untagResource`, `getCachedResponseMetadata`, `waiters`.
- * `ScanRequest` and `QueryRequest` calls currently only support EQ conditions.
+ * `ScanRequest` and `QueryRequest` calls currently only support EQ and GT conditions.
```

Comment:
are the various additions (i.e., this one and subsequent ones) of " and GT" to instances of "EQ" all appropriate/correct?

Reply:
I think so